frontend/packages/client/src/pages/About.js

```diff
@@ -19,12 +19,7 @@ const AboutPage = ({ location }) => {
   return (
     <div className={modal ? 'modal is-active' : undefined}>
       <div className="modal-background" onClick={closeModal} />
-      <div
-        className="modal-card rounded-sm px-4"
-        style={{
-          overflowY: 'scroll',
-        }}
-      >
```

Comment:
on wide screens `overflowY: 'scroll',` is adding  an empty scroll 

<img width="897" alt="Screen Shot 2022-07-26 at 23 49 22" src="https://user-images.githubusercontent.com/7526740/181150195-965d4df4-8e2a-4e2c-a581-f6ff4e8f752b.png">